

28/3/2022 16:32:37 => The property 'Medics.Clinicas' is of type 'ICollection<MedicoClinica>' which is not supported by the current database provider. Either change the property CLR type, or ignore the property using the '[NotMapped]' attribute or by using 'EntityTypeBuilder.Ignore' in 'OnModelCreating'.

28/3/2022 17:03:47 => The property 'Medics.Clinicas' is of type 'ICollection<MedicoClinica>' which is not supported by the current database provider. Either change the property CLR type, or ignore the property using the '[NotMapped]' attribute or by using 'EntityTypeBuilder.Ignore' in 'OnModelCreating'.

28/3/2022 17:13:31 => The property 'Medics.Clinicas' is of type 'ICollection<MedicoClinica>' which is not supported by the current database provider. Either change the property CLR type, or ignore the property using the '[NotMapped]' attribute or by using 'EntityTypeBuilder.Ignore' in 'OnModelCreating'.

29/3/2022 09:53:09 => The property 'Medics.Clinicas' is of type 'ICollection<MedicoClinica>' which is not supported by the current database provider. Either change the property CLR type, or ignore the property using the '[NotMapped]' attribute or by using 'EntityTypeBuilder.Ignore' in 'OnModelCreating'.

29/3/2022 10:17:59 => The property 'Medics.Clinicas' is of type 'ICollection<MedicoClinica>' which is not supported by the current database provider. Either change the property CLR type, or ignore the property using the '[NotMapped]' attribute or by using 'EntityTypeBuilder.Ignore' in 'OnModelCreating'.

29/3/2022 10:25:08 => The property 'Medics.Clinicas' is of type 'ICollection<MedicoClinica>' which is not supported by the current database provider. Either change the property CLR type, or ignore the property using the '[NotMapped]' attribute or by using 'EntityTypeBuilder.Ignore' in 'OnModelCreating'.

29/3/2022 11:04:52 => The property 'Medics.Clinicas' is of type 'ICollection<MedicoClinica>' which is not supported by the current database provider. Either change the property CLR type, or ignore the property using the '[NotMapped]' attribute or by using 'EntityTypeBuilder.Ignore' in 'OnModelCreating'.

29/3/2022 11:15:14 => The property 'Medics.Clinicas' is of type 'ICollection<MedicoClinica>' which is not supported by the current database provider. Either change the property CLR type, or ignore the property using the '[NotMapped]' attribute or by using 'EntityTypeBuilder.Ignore' in 'OnModelCreating'.

29/3/2022 11:30:44 => The property 'Medics.Clinicas' is of type 'ICollection<MedicoClinica>' which is not supported by the current database provider. Either change the property CLR type, or ignore the property using the '[NotMapped]' attribute or by using 'EntityTypeBuilder.Ignore' in 'OnModelCreating'.

29/3/2022 11:31:53 => The property 'Medics.Clinicas' is of type 'ICollection<MedicoClinica>' which is not supported by the current database provider. Either change the property CLR type, or ignore the property using the '[NotMapped]' attribute or by using 'EntityTypeBuilder.Ignore' in 'OnModelCreating'.

29/3/2022 11:45:44 => The property 'Medics.Clinicas' is of type 'ICollection<MedicoClinica>' which is not supported by the current database provider. Either change the property CLR type, or ignore the property using the '[NotMapped]' attribute or by using 'EntityTypeBuilder.Ignore' in 'OnModelCreating'.

29/3/2022 12:40:16 => Exception has been thrown by the target of an invocation.

29/3/2022 12:45:44 => Exception has been thrown by the target of an invocation.

29/3/2022 14:34:45 => Unable to resolve service for type 'Identidad.Api.Infraestructure.Services.IServices.IMedicsApplication' while attempting to activate 'Identidad.Api.Controllers.MedicsController'.

29/3/2022 17:56:10 => The method or operation is not implemented.

29/3/2022 18:02:03 => The method or operation is not implemented.

30/3/2022 09:56:58 => An error occurred while updating the entries. See the inner exception for details.

30/3/2022 10:13:12 => An error occurred while updating the entries. See the inner exception for details.

30/03/2022 14:35:46 => An error occurred while updating the entries. See the inner exception for details.

30/03/2022 14:53:29 => Unable to cast object of type 'System.Int32' to type 'System.Int64'.

30/03/2022 15:10:13 => An error occurred while updating the entries. See the inner exception for details.

30/03/2022 15:18:52 => An error occurred while updating the entries. See the inner exception for details.

30/03/2022 16:57:43 => An error occurred while updating the entries. See the inner exception for details.

30/03/2022 16:58:22 => An error occurred while updating the entries. See the inner exception for details.

30/03/2022 16:58:53 => An error occurred while updating the entries. See the inner exception for details.

31/03/2022 11:48:22 => Exception has been thrown by the target of an invocation.

31/03/2022 12:19:46 => An error occurred while updating the entries. See the inner exception for details.

31/03/2022 12:21:10 => An error occurred while updating the entries. See the inner exception for details.

01/04/2022 9:41:59 => Exception has been thrown by the target of an invocation.

04/04/2022 17:32:29 => Value cannot be null. (Parameter 'source')

04/04/2022 17:34:41 => Value cannot be null. (Parameter 'source')

04/04/2022 17:37:26 => Value cannot be null. (Parameter 'source')

04/04/2022 18:04:16 => Value cannot be null. (Parameter 'source')

04/04/2022 18:20:57 => Value cannot be null. (Parameter 'source')

04/04/2022 18:21:11 => Value cannot be null. (Parameter 'source')

04/04/2022 18:21:28 => Value cannot be null. (Parameter 'source')

06/04/2022 05:17:58 p. m. => A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)

08/04/2022 17:51:07 => Could not find a part of the path 'C:\Users\Lapaxsis-00\source\repos\API\Service.Catalog\wwwroot\layout\excel\indication\IndicacionesListado.xlsx'.

08/04/2022 17:51:23 => Could not find a part of the path 'C:\Users\Lapaxsis-00\source\repos\API\Service.Catalog\wwwroot\layout\excel\indication\IndicacionesFormulario.xlsx'.

08/04/2022 17:51:37 => Could not find a part of the path 'C:\Users\Lapaxsis-00\source\repos\API\Service.Catalog\wwwroot\layout\excel\indication\IndicacionesFormulario.xlsx'.

11/04/2022 16:16:39 => Could not find a part of the path 'C:\Users\Lapaxsis-00\source\repos\API\Service.Catalog\wwwroot\layout\excel\indication\IndicacionesListado.xlsx'.

11/04/2022 17:16:50 => Exception has been thrown by the target of an invocation.

11/04/2022 17:36:17 => Could not find a part of the path 'C:\Users\Lapaxsis-00\source\repos\API\Service.Catalog\wwwroot\layout\excel\indication\IndicacionesFormulario.xlsx'.

11/04/2022 18:36:25 => The instance of entity type 'MedicClinic' cannot be tracked because another instance with the same key value for {'MedicoId', 'ClinicaId'} is already being tracked. When attaching existing entities, ensure that only one entity instance with a given key value is attached. Consider using 'DbContextOptionsBuilder.EnableSensitiveDataLogging' to see the conflicting key values.

12/4/2022 07:47:20 => The value of 'MedicClinic.MedicoId' is unknown when attempting to save changes. This is because the property is also part of a foreign key for which the principal entity in the relationship is not known.

12/4/2022 07:53:38 => Value cannot be null. (Parameter 'source')

12/04/2022 10:05:47 => Value cannot be null. (Parameter 'source')

12/04/2022 10:20:10 => Value cannot be null. (Parameter 'source')

12/04/2022 11:08:52 => Value cannot be null. (Parameter 'source')

12/04/2022 11:10:08 => A connection was successfully established with the server, but then an error occurred during the pre-login handshake. (provider: SSL Provider, error: 0 - Tiempo de espera de la operación de espera agotado.)

12/04/2022 11:39:08 => Value cannot be null. (Parameter 'source')

12/04/2022 12:47:31 => Could not find a part of the path 'C:\Users\Lapaxsis-00\source\repos\API\Service.Catalog\wwwroot\layout\excel\indication\IndicacionesFormulario.xlsx'.

12/04/2022 12:57:18 => Could not find a part of the path 'C:\Users\Lapaxsis-00\source\repos\API\Service.Catalog\wwwroot\layout\excel\indication\IndicacionesListado.xlsx'.

13/4/2022 09:00:32 => Object reference not set to an instance of an object.

13/4/2022 09:00:39 => A connection was successfully established with the server, but then an error occurred during the pre-login handshake. (provider: SSL Provider, error: 0 - Tiempo de espera de la operación de espera agotado.)

13/04/2022 11:01:01 => Object reference not set to an instance of an object.

13/04/2022 13:12:02 => Unable to cast object of type 'System.Int64' to type 'System.Int32'.

13/04/2022 13:28:59 => Unable to cast object of type 'System.Int64' to type 'System.String'.

14/04/2022 10:55:49 a. m. => A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)

14/04/2022 10:57:22 a. m. => A network-related or instance-specific error occurred while establishing a connection to SQL Server. The server was not found or was not accessible. Verify that the instance name is correct and that SQL Server is configured to allow remote connections. (provider: Named Pipes Provider, error: 40 - Could not open a connection to SQL Server)

14/04/2022 02:19:19 p. m. => An error occurred while updating the entries. See the inner exception for details.

14/04/2022 02:20:39 p. m. => An error occurred while updating the entries. See the inner exception for details.

14/04/2022 02:48:58 p. m. => Object reference not set to an instance of an object.

14/04/2022 03:08:29 p. m. => The key value at position 0 of the call to 'DbSet<Branch>.Find' was of type 'string', which does not match the property type of 'Guid'.

14/04/2022 03:08:29 p. m. => The key value at position 0 of the call to 'DbSet<Branch>.Find' was of type 'string', which does not match the property type of 'Guid'.

14/04/2022 03:11:36 p. m. => Unable to cast object of type 'SelectListIterator`2[Service.Catalog.Domain.Study,Service.Catalog.Dtos.Study.StudyListDto]' to type 'System.Collections.Generic.List`1[Service.Catalog.Dtos.Study.StudyListDto]'.

14/04/2022 03:11:45 p. m. => Unable to cast object of type 'SelectListIterator`2[Service.Catalog.Domain.Study,Service.Catalog.Dtos.Study.StudyListDto]' to type 'System.Collections.Generic.List`1[Service.Catalog.Dtos.Study.StudyListDto]'.

14/04/2022 03:13:53 p. m. => There is already an open DataReader associated with this Connection which must be closed first.

14/04/2022 03:14:19 p. m. => There is already an open DataReader associated with this Connection which must be closed first.

14/04/2022 03:15:28 p. m. => There is already an open DataReader associated with this Connection which must be closed first.

14/04/2022 03:15:39 p. m. => There is already an open DataReader associated with this Connection which must be closed first.

14/04/2022 03:17:35 p. m. => There is already an open DataReader associated with this Connection which must be closed first.

14/04/2022 03:17:40 p. m. => There is already an open DataReader associated with this Connection which must be closed first.

14/04/2022 03:18:07 p. m. => There is already an open DataReader associated with this Connection which must be closed first.

14/04/2022 03:18:12 p. m. => There is already an open DataReader associated with this Connection which must be closed first.

14/04/2022 03:19:04 p. m. => There is already an open DataReader associated with this Connection which must be closed first.

14/04/2022 03:19:35 p. m. => There is already an open DataReader associated with this Connection which must be closed first.

14/04/2022 03:21:51 p. m. => There is already an open DataReader associated with this Connection which must be closed first.

14/04/2022 03:21:53 p. m. => There is already an open DataReader associated with this Connection which must be closed first.

14/04/2022 03:27:28 p. m. => Unable to cast object of type 'SelectListIterator`2[Service.Catalog.Domain.Study,Service.Catalog.Dtos.Study.StudyListDto]' to type 'System.Collections.Generic.List`1[Service.Catalog.Dtos.Study.StudyListDto]'.

14/04/2022 03:28:30 p. m. => Unable to cast object of type 'SelectListIterator`2[Service.Catalog.Domain.Study,Service.Catalog.Dtos.Study.StudyListDto]' to type 'System.Collections.Generic.List`1[Service.Catalog.Dtos.Study.StudyListDto]'.

14/04/2022 03:32:20 p. m. => Unable to cast object of type 'SelectListIterator`2[Service.Catalog.Domain.Study,Service.Catalog.Dtos.Study.StudyListDto]' to type 'System.Collections.Generic.List`1[Service.Catalog.Dtos.Study.StudyListDto]'.

14/04/2022 03:32:32 p. m. => Unable to cast object of type 'SelectListIterator`2[Service.Catalog.Domain.Study,Service.Catalog.Dtos.Study.StudyListDto]' to type 'System.Collections.Generic.List`1[Service.Catalog.Dtos.Study.StudyListDto]'.

14/04/2022 03:34:48 p. m. => Unable to cast object of type 'SelectListIterator`2[Service.Catalog.Domain.Study,Service.Catalog.Dtos.Study.StudyListDto]' to type 'System.Collections.Generic.List`1[Service.Catalog.Dtos.Study.StudyListDto]'.

14/04/2022 09:32:51 p. m. => The method or operation is not implemented.

14/04/2022 09:32:51 p. m. => The method or operation is not implemented.

14/04/2022 09:34:35 p. m. => The method or operation is not implemented.

14/04/2022 09:34:37 p. m. => The method or operation is not implemented.

18/04/2022 13:11:38 => The entity type 'Contact' requires a primary key to be defined. If you intended to use a keyless entity type, call 'HasNoKey' in 'OnModelCreating'. For more information on keyless entity types, see https://go.microsoft.com/fwlink/?linkid=2141943.

19/04/2022 16:58:22 => Value cannot be null. (Parameter 'source')

19/04/2022 16:59:10 => Value cannot be null. (Parameter 'source')

19/04/2022 17:56:46 => Object reference not set to an instance of an object.

19/04/2022 17:57:18 => Object reference not set to an instance of an object.

19/04/2022 17:58:36 => Object reference not set to an instance of an object.

20/04/2022 10:05:14 => Value cannot be null. (Parameter 'source')

20/04/2022 10:09:22 => Value cannot be null. (Parameter 'source')

20/04/2022 10:09:38 => Value cannot be null. (Parameter 'source')

20/04/2022 10:12:15 => Value cannot be null. (Parameter 'source')

20/04/2022 10:12:23 => Value cannot be null. (Parameter 'source')

20/04/2022 10:13:34 => Value cannot be null. (Parameter 'source')

20/04/2022 10:15:35 => Value cannot be null. (Parameter 'source')

20/04/2022 10:19:26 => The Include path 'Contacts->Compañia' results in a cycle. Cycles are not allowed in no-tracking queries; either use a tracking query or remove the cycle.

20/04/2022 10:22:57 => The Include path 'Contacts->Compañia' results in a cycle. Cycles are not allowed in no-tracking queries; either use a tracking query or remove the cycle.

20/04/2022 10:24:54 => Unable to cast object of type 'SelectListIterator`2[Service.Catalog.Domain.Company.Contact,Service.Catalog.Dtos.Company.ContactListDto]' to type 'System.Collections.Generic.ICollection`1[Service.Catalog.Dtos.Company.ContactListDto]'.

20/04/2022 13:24:28 => There isn't a worksheet named 'Compañias'.

20/04/2022 13:26:01 => Object reference not set to an instance of an object.

20/04/2022 13:40:33 => Object reference not set to an instance of an object.

20/04/2022 13:50:32 => Object reference not set to an instance of an object.

20/04/2022 14:57:14 => Object reference not set to an instance of an object.

22/04/2022 08:46:17 a. m. => The entity type 'TipoValor' requires a primary key to be defined. If you intended to use a keyless entity type, call 'HasNoKey' in 'OnModelCreating'. For more information on keyless entity types, see https://go.microsoft.com/fwlink/?linkid=2141943.
21/04/2022 11:09:00 => Unable to cast object of type 'System.Int32' to type 'System.Int64'.

21/04/2022 11:09:31 => Unable to cast object of type 'System.Int32' to type 'System.Int64'.

22/04/2022 11:47:17 => The MERGE statement conflicted with the FOREIGN KEY constraint "FK_CAT_CompañiaContacto_CAT_Compañia_CompañiaId". The conflict occurred in database "LAB_RAMOS_DEV_CATALOGO", table "dbo.CAT_Compañia", column 'Id'.
The statement has been terminated.

22/04/2022 11:50:29 => The MERGE statement conflicted with the FOREIGN KEY constraint "FK_CAT_CompañiaContacto_CAT_Compañia_CompañiaId". The conflict occurred in database "LAB_RAMOS_DEV_CATALOGO", table "dbo.CAT_Compañia", column 'Id'.
The statement has been terminated.

22/04/2022 15:22:16 => The ALTER TABLE statement conflicted with the FOREIGN KEY constraint "FK_CAT_Compañia_Provenance_ProcedenciaId". The conflict occurred in database "LAB_RAMOS_DEV_CATALOGO", table "dbo.Provenance", column 'Id'.

22/04/2022 15:25:54 => Invalid column name 'ProcedenciaId'.

22/04/2022 16:32:33 => Object reference not set to an instance of an object.

22/04/2022 16:57:11 => Object reference not set to an instance of an object.

22/04/2022 16:59:21 => Object reference not set to an instance of an object.

24/04/2022 21:56:28 => The process cannot access the file 'C:\Users\Lapaxsis-00\source\repos\API\Service.Catalog\wwwroot\layout\excel\company\CompañiaFormulario.xlsx' because it is being used by another process.